examples/plotting/plotting_blank_map.py

```diff
@@ -0,0 +1,54 @@
+  
+"""
+==============================================
+Plot Positions on a blank Map
+==============================================
+This example shows how to plot random positions on a blank map
+"""
+# Start by importing the necessary modules.
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import sunpy.map
+from sunpy.coordinates import frames
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+from astropy.io import fits
+
+################################################################################
+# Creating blank map data with zeros
+data = np.zeros((1000, 1000))
+
+# Define coordinates and frame of reference and make the header using sunpy.map.make_fitswcs_header
+skycoord = SkyCoord(0*u.arcsec, 0*u.arcsec, obstime = '2013-10-28', observer = 'earth', frame = frames.Helioprojective)
+header = sunpy.map.make_fitswcs_header(data, skycoord)
+
+# Use the sunpy.map.Map to make the blank map
+blank_map = sunpy.map.Map(data, header)
+
+################################################################################
+
+# Initialize the plot and add the map to it
+fig = plt.figure()
+ax = plt.subplot(projection=blank_map)
+im = blank_map.plot()
+
+# Prevent the image from being re-scaled while overplotting.
+ax.set_autoscale_on(False)
+
+# Coordinates that are being plotted - (0, 0), (50, 100) and (400, 400)
+xc = [0,50,400] * u.arcsec
+yc = [0,100,400] * u.arcsec
+
+# Place and mark coordinates on the plot
+coords = SkyCoord(xc, yc, frame=blank_map.coordinate_frame)
+p = ax.plot_coord(coords, 'o')
+
+# Set title
+ax.set_title('Custom plot with WCSAxes')
+
+# Add the colorbar and display the plot
+plt.colorbar()
```

Comment:
Why add a colorbar?

Reply:
It's just plain black without a colorbar, would that be fine?

Reply:
Well its a blank map, why would it need a colorbar? 

Also if it's all 0s, it should made to be a white map? I am not sure if I prefer it to be a black or white map.

Reply:
If it's a zero then it corresponds to a black. I'll remove the colorbar.

Reply:
I assume we could rescale the map so it was white instead?

Reply:
I am not able to figure this out, could you please help me?

Reply:
You mean changing the scale to white? 

I am not really sure how to do that, https://docs.sunpy.org/en/stable/generated/gallery/plotting/map_editcolormap.html#sphx-glr-generated-gallery-plotting-map-editcolormap-py might offer some insight to changing the norm or color map to acheive this. 

Otherwise we just leave it black.

Reply:
if you set alpha=0 in `plot()` it should be white.